.gitmodules

```diff
@@ -48,13 +48,13 @@
 	branch = master
 [submodule "third-party/tray"]
 	path = third-party/tray
-	url = https://github.com/dmikushin/tray
-	branch = master
+	url = https://github.com/LizardByte/tray
+	branch = balloons
```

Comment:
Should we merge to master rather than leaving the branch here?

Reply:
Additionally, would the changes made be compatible with the upstream project goals? The few interactions I've had with them, they seemed pretty receptive to improvements.

Reply:
I wasn't thinking in terms of upstreaming. I was thinking we probably want a protected branch that can't be directly committed to.

Reply:
That repo doesn't have branch protection rules as of yet... I can set it up whenever though.

Reply:
For upstreaming the only problem is feature parity, since I haven't touched the macOS side of things. As for the branch @ReenigneArcher let me know if I should make it myself or if you're planning to make it

Reply:
`master` branch is now protected and made another branch `master_upstream` if we eventually want to upstream the changes.

It will require a pull request to merge anything into `master`, but there is no CI.

Reply:
I've updated our fork, merging our changes to the master branch